eng/SnippetGenerator/Program.cs

```diff
@@ -20,7 +20,7 @@ public void OnExecuteAsync()
         {
             var baseDirectory = new DirectoryInfo(BasePath).Name;
             var baseDirParent = Directory.GetParent(BasePath).Name;
```

Comment:
is this variable used?

Reply:
No, I'll remove that. But let's continue that review on the other PR. :) I'll make the change there. I can rebase this one later to effectively pick it up.